doc/source/cluster/kubernetes/examples/image-resize-example.md

```diff
@@ -0,0 +1,96 @@
+(kuberay-image-resize-example)=
+
+# Image Resize Example

```

Comment:
Might be good to reference "Ray Data" somewhere in the title as that's the primary library used for this example with kuberay